knife/lib/chef/knife/client_bulk_delete.rb

```diff
@@ -0,0 +1,104 @@
+#
```

Comment:
This should have been a rename-then-change, not sure why this and others are showing new.  THat'l make this more challenging to review...